base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/blocks/EditBlockDialog.java

```diff
@@ -85,6 +100,10 @@ public EditBlockDialog(Shell parentShell, EditableBlock block, EditableConfigura
 		for (ErrorMessageProvider provider : viewModels) {
 			provider.addPropertyChangeListener("error", errorListener);
         }
+
+		for (ErrorMessageProvider provider : viewModels) {
+            provider.addPropertyChangeListener("warning", warningListener);
+        }
```

Comment:
This loop is exactly the same as the one above it, it would be neater to add both listeners in the same loop.